### ISSUE TITLE
accuracy dropped a lot  when converting rtmpose onnx file to tensorrt format 

### ISSUE THREAD

**Issue by Xinxinatg** [open-mmlab/mmpose#2579]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmpose/issues) and [Discussions](https://github.com/open-mmlab/mmpose/discussions) but cannot get the expected help.
- [X] The bug has not been fixed in the latest version(https://github.com/open-mmlab/mmpose).

### Environment
First converting the pytorch model to onnx then converting the onnx model to tensorrt format within triton inference server.

### Reproduces the problem - code sample

        import argparse
        
        import torch
        import torch.nn as nn
        import onnx
        
        from mmpose.apis import init_model
        
        
        class RTMPoseWithDecode(nn.Module):
            def __init__(self, config, checkpoint):
                super().__init__()
                self.detector = init_model(config, checkpoint, 'cpu')
                self.mean = torch.tensor([123.675, 116.28, 103.53])
                self.std = torch.tensor([58.395, 57.12, 57.375])
        
            def forward(self, x):
                x[0][0][:4] = x[0][1][:4]
                resized_img= (x - self.mean[:, None, None]) / self.std[:, None, None]
                simcc_x, simcc_y = self.detector.forward(resized_img, None)
        
                max_val_x, x_locs = torch.max(simcc_x, dim=2)
                max_val_y, y_locs = torch.max(simcc_y, dim=2)
                scores = torch.maximum(max_val_x, max_val_y)
                keypoints = torch.stack([x_locs, y_locs], dim=-1)
                keypoints = keypoints.float() / self.detector.cfg.codec.simcc_split_ratio
        
                tensor_b = scores.unsqueeze(-1)  # Now tensor_b has shape (1, 17, 1)
        
                combined_tensor = torch.cat((keypoints, tensor_b), dim=-1)  # Now combined_tensor has shape (1, 17, 3)
        
                return combined_tensor
        
        input_size=[192, 256]
        
        model = RTMPoseWithDecode("projects/rtmpose/rtmpose/body_2d_keypoint/rtmpose-l_8xb256-420e_coco-256x192.py", "rtmpose-l_simcc-aic-coco_pt-aic-coco_420e-256x192-f016ffe0_20230126.pth")
        
        dummy_image = torch.zeros((1, 3, *input_size[::-1]), device='cpu')
        
        torch.onnx.export(
            model,
            dummy_image,
            "/content/output.onnx",
            input_names=['input'],
            dynamic_axes={'input': {
                0: 'batch'
            }})
        

### Reproduces the problem - command or script

I followed the instructions [here](https://zhuanlan.zhihu.com/p/632466141) to integrate the postprocess into onnx model. Those are the [onnx file](https://1drv.ms/u/s!Au4pjft__tnygvUv2yR7Buc9-A61kQ?e=x6ZO5N) and the [output videos](https://1drv.ms/v/s!Au4pjft__tnygvUuZqW1mZxh3iBTYw?e=nKgvUe) accordingly. From the output you can find that some keypoints out of the 17 are constantly located at the top left of the bounding box, while the rest are accurate enough.

### Reproduces the problem - error message

The errors are in the output video

### Additional information

The model is from rtmpose-l_8xb256-420e_coco-256x192.py

**Comment by Tau-J** [comment]
Thanks for using MMPose. I suggest that you can do some debugging to locate the error. We cannot figure out the reason from the information you provide.

**Comment by Xinxinatg** [comment]
> Thanks for using MMPose. I suggest that you can do some debugging to locate the error. We cannot figure out the reason from the information you provide.

Thanks, it was probably caused by the different quantization method in onnx runtime and tensorrt as per https://github.com/NVIDIA/TensorRT/issues/1568.
